HCK07_2020_Seattle/projects/PROJECTS.md

```diff
@@ -2,21 +2,21 @@
 
 The lists below provides an overview of specific projects for which at least one or more attendees have signed up for to contribute to. Projects are collaborative in nature. If you are attending the hackathon and are interested in participating in any of the existing projets then please contact the team listed on the project page and add your name to the list of participants for the project.
 
-* [Template Project](projects/template)
+* [Template Project](template)
 
-* [Add Support and CI for NWB Extensions](projects/ExtensionsCI)
-* [Add Dimension Scales/Coordinates in MatNWB](projects/DimScalesMatlab)
-* [Integrate Zarr I/O backend with HDMF/PyNWB](projects/zarrio)
-* [Evaluate, refine, and review ICEPhys extension proposal](projects/icephys_extension)
-* [Performance Benchmarks and Profiling for PyNWB](projects/benchmark_profile)
-* [Simplify PyNWB interface](projects/DeBoilerPlateNWB)
-* [NWB-Datajoint Integration](projects/NWB_DataJoint_Integration)
-* [Open Ephys NWB Writer](projects/Open_Ephys_NWB_Writer)
-* [Convert and add NWB2 datasets to DANDI](projects/NWB2.0Datasets)
-* [OpenScope2NWB](projects/OpenScope2NWB)
-* [Update nwb.org website](projects/update_nwb_website)
-* [Convert, annotate, analyze, and co-cluster intracellular electrophysiology datasets collected across labs](projects/icephys_meta_analysis)
-* [Visualization of hierarchical icephys experiments](projects/icephys_extension_widget)
+* [Add Support and CI for NWB Extensions](ExtensionsCI)
+* [Add Dimension Scales/Coordinates in MatNWB](DimScalesMatlab)
+* [Integrate Zarr I/O backend with HDMF/PyNWB](zarrio)
+* [Evaluate, refine, and review ICEPhys extension proposal](icephys_extension)
+* [Performance Benchmarks and Profiling for PyNWB](benchmark_profile)
+* [Simplify PyNWB interface](DeBoilerPlateNWB)
+* [NWB-Datajoint Integration](NWB_DataJoint_Integration)
+* [Open Ephys NWB Writer](Open_Ephys_NWB_Writer)
+* [Convert and add NWB2 datasets to DANDI](NWB2.0Datasets)
+* [OpenScope2NWB](OpenScope2NWB)
+* [Update nwb.org website](update_nwb_website)
+* [Convert, annotate, analyze, and co-cluster intracellular electrophysiology datasets collected across labs](icephys_meta_analysis)
+* [Visualization of hierarchical icephys experiments](icephys_extension_widget)

```

Comment:
I believe this will break the links on the main page. I'll build the branch on my machine with Jekyll to test.

Reply:
So this only works in https://neurodatawithoutborders.github.io/nwb_hackathons/ but not with clicking in github itself.? In that case we can close the PR as I wanted to get the links fixed inside github but that seems to break it for jekyll.

Reply:
I think the issue is that in GitHub the links are relative to the ``projects/ `` folder and for Jekyll the file is included in the main README.md which is in the parent folder. I think the simplest fix may be to move the PROJECTS.ms and BREAKOUTS.md file to the parent folder and then I think the links needed for Jekyll should be the same as on GitHub itself.

Reply:
Closing as not-a-problem.